doc/html/src/examples.html

```diff
@@ -187,6 +187,8 @@ <h1>A Series of Example Programs</h1>
 
 <li><L1><a href="examples/miscellaneous_ex14.html">Infinite Elements With Imaginary Frequency</a></L1></li>
 
+<li><L1><a href="examples/miscellaneous_ex10.html">Infinite Elements With Zero Frequency: Unbound Laplace Equation</a></L1></li>
```

Comment:
Not sure how I missed this before - this should be ex15, no?

Reply:
thanks. Sure this is wrong.